manifests/init.pp

```diff
@@ -31,8 +34,10 @@
   $make_install   = true,
   $node_path      = '/usr/local/node/node-default/lib/node_modules',
   $python_package = 'python',
+  $use_cpus       = $::processorcount,
```

Comment:
`use_cpus` might be a bit confusing as it sounds like a `bool` parameter. I suggest something like `$cpu_cores`.